primitives/io/src/lib.rs

```diff
@@ -984,6 +1000,7 @@ pub type SubstrateHostFunctions = (
 	logging::HostFunctions,
 	sandbox::HostFunctions,
 	crate::trie::HostFunctions,
+	offchain_index::HostFunctions,
```

Comment:
I don't get what is happening here, but I can't imagine `offchain_index::HostFunctions,` twice in the list to be correct.

Reply:
huh, where do you see it twice?

Reply:
I am losing my eyesight, ignore :D

Reply:
the only comment would be why not merge this with `offchain::HostFunctions`? won't they always be used and available together?

Reply:
Not necessarily, offchain workers can exist without the indexing API - in reality I don't know how correlated the usage will be.